x-pack/plugins/apm/public/components/shared/PropertiesTable/index.tsx

```diff
@@ -43,21 +44,41 @@ const EuiIconWithSpace = styled(EuiIcon)`
   margin-right: ${px(units.half)};
 `;
 
-export function getPropertyTabNames(selected: string[]): string[] {
+export interface Tab {
+  key: string;
+  label: string;
+}
+
+export function getPropertyTabNames(selected: string[]): Tab[] {
   return PROPERTY_CONFIG.filter(
     ({ key, required }: { key: string; required: boolean }) =>
       required || selected.includes(key)
-  ).map(({ key }: { key: string }) => key);
+  ).map(({ key, label }: Tab) => ({ key, label }));
 }
 
 function getAgentFeatureText(featureName: string) {
   switch (featureName) {
     case 'user':
-      return 'You can configure your agent to add contextual information about your users.';
```

Comment:
Just curious: why not use `i18n.translate(...)` here. Would look cleaner than wrapping in `FormattedMessage`.

Reply:
Yes, it might be weird. We try to use `FormattedMessage` where `jsx` is expected. When a string is required we use `intl.formatMessage`. And in other case (e.g. to translate a label that is located out of a component) we use `i18n.translate`

Reply:
Is it not possible to use `i18n.translate`? I think it would be nicer not having to pass `intl` in as an argument.